file_paths.md

```diff
@@ -152,7 +152,7 @@ The files annotated by an '*' indicate that they are created by package installa
 
 On recent versions of Windows, e.g. 2008 & 2012, the installation path
 defaults to `C:\Program Files\Puppet Labs` and the common app data
-directory defaults to `C:\ProgramData`. On 2003, common app data is
+directory defaults to `C:\ProgramData\PuppetLabs`. On 2003, common app data is
```

Comment:
Nit, "common app data" is a special windows specific folder and is `C:\ProgramData`, and we add our stuff within `PuppetLabs`.